lib/panomosity/xlsx_writer.rb

```diff
@@ -0,0 +1,51 @@
+require 'write_xlsx'
+
+module Panomosity
+  module XLSX_Writer
```

Comment:
If you want to be cool this could be a class instead so you can use instance variables and abstract out logic without cluttering other classes with unusable methods.

```ruby
class XLSXWriter
  attr_accessor :input_files, :output_file, :workbook

  def initialize(*input_files, output:)
    @input_files = input_files
    @output_file = output_file
    @workbook = WriteXLSX.new("#{output_file}.xlsx")
  end

  def save
    input_files.each do |input_file|
      write_worksheet(input_file|)
    end

    workbook.close
  end

  private

  def write_worksheet(input_file)
    workbook.add_worksheet(input_file
    # etc.
  end
end 

writer = XLSXWriter.new('aaaa.csv', 'bbbb.csv', output: 'aabb')
writer.save
```